src/hotspot/cpu/riscv/stubDeclarations_riscv.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights
+ * reserved.  Copyright (c) 2024, Red Hat, Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+#ifndef CPU_RISCV_STUBDECLARATIONS_HPP
+#define CPU_RISCV_STUBDECLARATIONS_HPP
+
+#define STUBGEN_INITIAL_BLOBS_ARCH_DO(do_stub,                          \
+                                      repeat_stub,                      \
+                                      do_arch_blob,                     \
+                                      do_arch_entry,                    \
+                                      do_arch_entry_init)               \
+  do_arch_blob(initial, 10000)                                          \
+  do_arch_entry(riscv, initial, get_previous_sp,                        \
+                get_previous_sp_entry,                                  \
+                get_previous_sp_entry)                                  \
```

Comment:
`get_previous_sp_entry` has been removed by https://bugs.openjdk.org/browse/JDK-8344265.

Reply:
Thanks for spotting this. I missed it when reconciling against your deletion. I think I got lal the others though.

Fixed in latest push.